GUIDE.md

```diff
@@ -42,7 +42,7 @@ As `$USER` user with `sudo` privileges,
 
         [$USER@assisted_installer assisted-test-infra]# make create_full_environment
 
-1.  **(Optional)** Currently the Installer defaults to deploying OpenShift 4.5. If you wish to change this value, set `OPENSHIFT_VERSION` to a different value, e.g. 4.6
+1.  **(Optional)** Currently the Installer defaults to deploying OpenShift 4.6. If you wish to change this value, set `OPENSHIFT_VERSION` to a different value, e.g. 4.7
```

Comment:
Not entirely sure about this one. Is 4.7 even currently supported? Should consider removing this sentence altogether if we don't.

Reply:
Yes it does.